app/views/content/get-school-experience/get-school-experience.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Get school experience
+image: "/assets/images/international-dt.jpg"
+mobileimage: "/assets/images/steps-hero-mob.jpg"
+description: |-
+  School experience can help your teacher training application. Here's how to get school experience.
+date: "2021-05-04"
+right_column:
+  ctas:
+    - title: Find school experience
+      link_text: "Request early years, primary or secondary experience."
+      link_target: "https://schoolexperience.education.gov.uk/"
+      icon: "icon-person"
+      hide_on_mobile: Yes
+      hide_on_tablet: Yes
+keywords:
+  - School experience
+  - Teaching experience
+  - Teacher experience
+---
+
+School experience can help you decide whether teaching is right for you.
+
+Spending some time in a school allows you to:
+
+* observe lessons and other teacher responsibilities
+* talk to teachers
+* get an insight into day-to-day school life
+
+It can also really help [your teacher training application](/tips-on-applying-for-teacher-training), although it’s not a training entry requirement.
+
+To find experience you can:
+
+* [request early years, primary or secondary experience on GOV.UK](https://schoolexperience.education.gov.uk/) - not all schools advertise experience here and some only offer online experience due to coronavirus (COVID-19) restrictions
+* approach schools yourself
+
+## Approach schools yourself
+
+[Find schools near you](https://get-information-schools.service.gov.uk/) and give them a ring to find out who to talk to about getting school experience.
+
+It's normal to have to try a few schools before you find one that can help.
```

Comment:
Similar to Pete's query below - are these two lines not combine-able? They feel v similar.